enterprise/api_client/enterprise_catalog.py

```diff
@@ -100,6 +103,39 @@ def delete_enterprise_catalog(self, catalog_uuid):
             )
             return {}
 
+    @JwtLmsApiClient.refresh_token
+    def get_content_metadata(self, enterprise_customer, enterprise_catalogs=None):
+        """
+        Return all content metadata contained in the catalogs associated with the EnterpriseCustomer.
+
+        Arguments:
+            enterprise_customer (EnterpriseCustomer): The EnterpriseCustomer to return content metadata for.
+            enterprise_catalogs (EnterpriseCustomerCatalog): Optional list of EnterpriseCustomerCatalog objects.
+
+        Returns:
+            list: List of dicts containing content metadata.
+        """
+        content_metadata = OrderedDict()
+        enterprise_customer_catalogs = enterprise_catalogs or enterprise_customer.enterprise_customer_catalogs.all()
+
+        for enterprise_customer_catalog in enterprise_customer_catalogs:
+            catalog_uuid = enterprise_customer_catalog.uuid
+            endpoint = getattr(self.client, self.GET_CONTENT_METADATA_ENDPOINT.format(catalog_uuid))
+            try:
+                response = endpoint.get()
```

Comment:
this might come later, but one outstanding item is that these endpoints should be paginating results, but it hasn't happened yet so this is fine for now. once it does, we may need to add a parameter to make sure we're getting all the results at once.

Reply:
👍